app/.gitignore

```diff
@@ -1 +1,94 @@
-/build
+# Built application files
```

Comment:
This file should not be changed since we already have a .gitignore file within the main directory wathub/.gitignore